packages/mds-compliance/tests/engine-test.spec.ts

```diff
@@ -140,38 +140,37 @@ describe('Tests Compliance Engine', () => {
     done()
   })
 
-  /* TODO -- Implement Speed Compliance */
-  // it('Verifies speed compliance violation', done => {
-  //   const devices = makeDevices(5, now())
-  //   const events = makeEventsWithTelemetry(devices, now(), CITY_OF_LA, 'trip_start', 500)
-  //   test.assert.doesNotThrow(() => validatePolicies(policies))
-  //   test.assert.doesNotThrow(() => validateGeographies(geographies))
-  //   test.assert.doesNotThrow(() => validateEvents(events))
+  it('Verifies speed compliance violation', done => {
+    const devices = makeDevices(5, now())
+    const events = makeEventsWithTelemetry(devices, now(), CITY_OF_LA, 'trip_start', 500)
+    test.assert.doesNotThrow(() => validatePolicies(policies))
+    test.assert.doesNotThrow(() => validateGeographies(geographies))
+    test.assert.doesNotThrow(() => validateEvents(events))
 
-  //   const filteredEvents = filterEvents(events)
-  //   const filteredPolicies = filterPolicies(policies)
-  //   const deviceMap: { [d: string]: Device } = devices.reduce(
-  //     (deviceMapAcc: { [d: string]: Device }, device: Device) => {
-  //       return Object.assign(deviceMapAcc, { [device.device_id]: device })
-  //     },
-  //     {}
-  //   )
-  //   const results = filteredPolicies.map(policy => processPolicy(policy, filteredEvents, geographies, deviceMap))
-  //   results.forEach(result => {
-  //     if (result) {
-  //       result.compliance.forEach(compliance => {
-  //         if (
-  //           compliance.rule.geographies.includes(CITY_OF_LA) &&
-  //           compliance.matches &&
-  //           compliance.rule.rule_type === RULE_TYPES.speed
-  //         ) {
-  //           test.assert(compliance.matches.length !== 0)
-  //         }
-  //       })
-  //     }
-  //   })
-  //   done()
-  // })
+    const filteredEvents = filterEvents(events)
+    const filteredPolicies = filterPolicies(policies)
+    const deviceMap: { [d: string]: Device } = devices.reduce(
+      (deviceMapAcc: { [d: string]: Device }, device: Device) => {
+        return Object.assign(deviceMapAcc, { [device.device_id]: device })
+      },
+      {}
+    )
+    const results = filteredPolicies.map(policy => processPolicy(policy, filteredEvents, geographies, deviceMap))
+    results.forEach(result => {
+      if (result) {
+        result.compliance.forEach(compliance => {
+          if (
+            compliance.rule.geographies.includes(CITY_OF_LA) &&
+            compliance.matches &&
+            compliance.rule.rule_type === RULE_TYPES.speed
+          ) {
+            test.assert(compliance.matches.length !== 0)
```

Comment:
Test should be updated to check the `total_violations` param, and there should be 5 of them